lesson_2/series.py

```diff
@@ -0,0 +1,40 @@
+"""
+Problem 3: fibonacci, lucas, sum_series
+
+"""
+
+def fibonacci(n):
+    """return the nth value in a fibonacci series (starting with 0 index)"""
+    if n == 0:
+        return(0)
+    elif n == 1:
+        return(1)
+    else:
+        return fibonacci(n-1) + fibonacci(n-2)
+
+
+def lucas(n):
+    """return the nth value in a lucas series (starting with 0 index). Lucas series begins with 2,1,3,..."""
+    if n == 0:
+        return(2)
+    elif n == 1:
+        return(1)
+    else: 
+        return lucas(n-1) + lucas(n-2)
+
+def sum_series(n, x = 0, y = 1):
```

Comment:
minor pep8 style violation: no need for spaces around `=` sign -> `(n, x=0, y=1)`